.github/workflows/bulk-dependency-updates.yaml

```diff
@@ -119,10 +118,10 @@ jobs:
           if [ -z "$PR" ]; then
             gh pr create \
             --head "$BRANCH_NAME" \
-            --title "Automated dependency upgrades - dispatch variant" \
+            --title "Automated dependency upgrades" \
             --reviewer "$reviewers" \
             --label "dependencies" \
-            --body "Full log: https://github.com/hashicorp/$REPO/actions/runs/$RUN_ID
```

Comment:
That reminds me we also need to change some code on the vault side to fix this: https://github.com/hashicorp/vault/blob/c6ef0800a00f3634b869e1152cfbf5581753633f/.github/workflows/plugin-update-check.yml#L47